forum/qa-plugin/report-reason/q2apro-flag-reasons-layer.php

```diff
@@ -72,42 +72,16 @@ public function body_hidden()
                 . qa_lang('q2apro_flagreasons_lang/reason')
                 . '
                         </h4>
-                        <label>
-                            <input type="radio" name="qa-spam-reason-radio" value="1" checked>
-                            <span>'
-                . q2apro_flag_reasonid_to_readable(1)
-                . '</span>
-                        </label>
-                        <label>
-                            <input type="radio" name="qa-spam-reason-radio" value="2">
-                            <span>'
-                . q2apro_flag_reasonid_to_readable(2)
-                . '</span>
-                        </label>
-                        <label>
-                            <input type="radio" name="qa-spam-reason-radio" value="3">
-                            <span>'
-                . q2apro_flag_reasonid_to_readable(3)
-                . '</span>
-                        </label>
-                        <label>
-                            <input type="radio" name="qa-spam-reason-radio" value="4">
-                            <span>'
-                . q2apro_flag_reasonid_to_readable(4)
-                . '</span>
-                        </label>
-                        <label>
-                            <input type="radio" name="qa-spam-reason-radio" value="5">
-                            <span>'
-                . q2apro_flag_reasonid_to_readable(5)
-                . '</span>
-                        </label>
-                        <label>
-                            <input type="radio" name="qa-spam-reason-radio" value="6">
+                        ';
+                 for($i=0;$i<=6;$i++){
```

Comment:
`for` niczego nie rozwiązuje, bo gdy pojawi się nowy powód również będę musiał to zmieniać. Wg mnie byłoby lepiej, jakby powody były gdzieś w jednej tablicy wspólnej dla całego pluginu i na jej podstawie aby automatycznie wszystko się dobierało.
Ponadto sam for nie przestrzega PSR-2, wcięcia są kompletnie rozsypane, a nazwa zmiennej `$i` nic nie mówi :P

Reply:
`i` - iteracja :P